src/player.cpp

```diff
@@ -4942,9 +4949,12 @@ void player::update_stamina( int turns )
 {
     float stamina_recovery = 0.0f;
     // Recover some stamina every turn.
-    if( !has_effect( effect_winded ) ) {
-        // But mouth encumberance interferes.
-        stamina_recovery += std::max( 1.0f, 10.0f - ( encumb( bp_mouth ) / 10.0f ) );
+    // Mutated stamina works even when winded
+    float stamina_multiplier = ( !has_effect( effect_winded ) ? 1.0f : 0.0f ) +
```

Comment:
This ternary with the negated condition is slightly confusing. It would be cleared without the negation (and inverted results) as `has_effect() ? 0 : 1`.